tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md

```diff
@@ -273,9 +273,9 @@ In this tutorial, you learned how to create a custom pallet by defining storage,
 
 To review this implementation, you can find the complete pallet code below:
 
-???+ example "Complete Pallet Code"
+???- code "src/lib.rs"
```

Comment:
This might seem like a weird comment, but I don't like using a leading sentence with a colon that points to a snippet that is hidden by default.

I'd suggest one of the following:
- `To review this implementation, you can find the complete pallet code below.`
- `To review this implementation, you can find the complete pallet code below. Click to expand and view the code.`
- Keeping the leading sentence as is (`To review this implementation, you can find the complete pallet code below:`)  and expanding the code element by default